lib/components/CentralDevice.jsx

```diff
@@ -177,6 +177,11 @@ const CentralDevice = ({
                         key="deleteBondInfo"
                         title="Delete bond information"
                         onSelect={onDeleteBondInfo}
+                        disabled={
+                            security === null
+                                ? true
+                                : security.bondStore.size === 0
+                        }
```

Comment:
This could be simplified to ```disabled={security === null || security.bondStore.size === 0}```, but this is not a big enough change that I would suggest updating the PR. So LGTM 👍